circle.yml

```diff
@@ -1,26 +0,0 @@
-machine:
```

Comment:
Why do you remove this file? It is part of the upstream.

Reply:
If we ever submit PRs to the upstream, we should put it back, yes. Meanwhile, if we don't use it in our master, why have it there at all?

Same thing for the terragrunt executable, we were only keeping it because it is in the upstream but it actually caused problems with the deployment when building the project.

Reply:
Agreed!

Reply:
Especially for the terragrunt executable. It has nothing to do there.